packages/terra-i18n/lib/translations/en-US.js

```diff
@@ -11,7 +11,12 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
 (0, _reactIntl.addLocaleData)(_en2.default);
 
 var messages = {
-  'Terra.ajax.error': 'This content failed to load.'
+  'Terra.ajax.error': 'This content failed to load.',
+  'Terra.demographicsBanner.dateOfBirth': 'DOB: ',
```

Comment:
I wasn't 100% certain on our best solution. Here is how blue_steel handles those concerns

    %b= dob
    - if deceased
      = "#{t('blue_steel.demographics.deceased')}:"
      %b= dod
    - if ga
      = "#{t('blue_steel.demographics.ga')}:"

Not sure 100% if it's best to keep the : as part of the message, or to include it in the markup of the application.